docs/installation.md

```diff
@@ -313,23 +345,6 @@ Mill have scalafmt support built-in:
 
 - [scalafmt module](http://www.lihaoyi.com/mill/page/configuring-mill.html#reformatting-your-code)
 
-## Vim
```

Comment:
I don't know if this installation method is still working so  removed it in favor of Metals.